src/components/right_page/Gallery.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+import deptEvent from "../../assets/img/dept_event.png";
+import deptMkt from "../../assets/img/dept_mkt.png";
+import deptPr from "../../assets/img/dept_pr.png";
+import deptTech from "../../assets/img/dept_tech.png";
+
+//padding and width for image should be adjusted when replacing image source
+export default function Gallery(props){
+    return(
+        <div className="gallery-container">
```

Comment:
You can add a bootstrap class `container` here so it creates horizontal padding and controls width.